lib/include/ultrahdr/editorhelper.h

```diff
@@ -1,44 +0,0 @@
-/*
- * Copyright 2024 The Android Open Source Project
```

Comment:
I would recommend that we keep this file and keep ultrahdrcommon.h simple

Reply:
Done